src/main/java/com/lesson7/solution/controller/TodoController.java

```diff
@@ -0,0 +1,52 @@
+package com.lesson7.solution.controller;
+
+import com.lesson7.solution.model.TaskHistoryResponseDto;
+import com.lesson7.solution.model.TodoCreateDto;
+import com.lesson7.solution.model.TodoResponseDto;
+import com.lesson7.solution.model.TodoUpdateDto;
+import com.lesson7.solution.service.TaskHistoryService;
+import com.lesson7.solution.service.TodoService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/todos")
+public class TodoController {
+    private final TodoService todoService;
+    private final TaskHistoryService taskHistoryService;
+
+    @PostMapping
+    public TodoResponseDto create(@RequestBody TodoCreateDto todoCreateDto) {
```

Comment:
Don't forget to use `@Valid`. Without this annotation validation will not work
```suggestion
    public TodoResponseDto create(@RequestBody @Valid TodoCreateDto todoCreateDto) {
```